streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -142,39 +142,31 @@ public StreamTask(final TaskId id,
         log.info("{} Initializing state stores", logPrefix);
         initializeStateStores();
         stateMgr.registerGlobalStateStores(topology.globalStateStores());
-        initTopology();
+        init();
```

Comment:
nit: could we move the `initializeStateStores` as well as global state store registration inside `init` functions for both streamtask and standby task, since we are closing them inside `close()` as well?